Infrastructure-Scripts/Get-EnvironmentDeploymentRecords.ps1

```diff
@@ -0,0 +1,62 @@
+<#
```

Comment:
Not sure about the script name `Get-EnvironmentDeploymentRecords`, if I execute this I expect it to return some records. The intention behind this script is a bit different and I think it could do with a rename.

Reply:
Name of script has been updated to Wait-AzureDevOpsDeployment